src/index.mako.html

```diff
@@ -187,7 +187,7 @@
 
 % endif
     <div id="loader"></div>
-    <div tabindex="1" ga-map ga-map-map="map">
+    <div tabindex="1" ga-map ga-map-map="map" ga-map-ol3d="ol3d">
```

Comment:
I know there are currently no one-time binding expressions in `index.mako.html`, but you could use one here for `ol3d` (`ga-map-ol3d="::ol3d"`).

@gjn, would it make sense to start using one-time binding expressions to minimize the number of watchers?